lib/improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -854,8 +934,7 @@ def _check_unit_compatibility(mean_values, variance_values,
                    'not equivalent/compatible.'.format(err))
             raise ValueError(msg)
 
-    @staticmethod
-    def _mean_and_variance_to_probabilities(mean_values, variance_values,
+    def _mean_and_variance_to_probabilities(self, mean_values, variance_values,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method name "_mean_and_variance_to_probabilities" doesn't conform to '((test_[a-zA-Z0-9_]+)|([a-z_][a-z0-9_]{2,30}))$' pattern](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4540588)